examples/splunk-enterprise-index-routing/README.md

```diff
@@ -0,0 +1,5 @@
+# Example of chart configuration
+
+## Route log records to specific Splunk Enterprise indexes
+Configure log collection to set the Splunk index to target with logs to the name
+of the kubernetes namespace they originate from.
```

Comment:
I don't understand this what this sentence is saying. I'm kind of lost on "set the Splunk index to target with logs to the name". Could you explain what's meant here?